### ISSUE TITLE
LKS Issues Reopen by Mr. Viswanathan on 20-06-2023

### ISSUE THREAD

**Issue by TabrejAlam3K** [3ktech/General#548]

**Comment by TabrejAlam3K** [comment]
This are the below points reopen by Mr. Viswanathan Sir.

1. #271
2. #282 
3. #258 
4. #218 
5. #264 
6. #240 
7. #262 
8. #234

**Comment by TabrejAlam3K** [comment]
1. Check Print - Need to sit with the client (Ms. Sandhya) to fix it. **(Pending)**
2. Application Un-Application - This is working fine in Production as well. - **Working**
3. Permission Issues - This is working in UAT but not in Production. - Move the Changes from UAT to Prod
4. Report 50105 - Under Re-Testing
5. Report 50157 In Favor of - this is working for only the Currency INR, Need to check for other currency. - **(Pending)**
6. Check Voiding - Pending by Technical End. -**(Pending)**
7. Branch GST Invoice logo - Under Re-Testing
8. Exchange Rate - The field have a maximum of 6 decimal places client till 7 decimal Places. **(New Requirement)**

**Comment by TabrejAlam3K** [comment]
Hi Ms. @audhi3k Mam - We need to close the this ticket also, Please align any technical for this.

1. Permission Set - I need to filter the Location based on the User Setup Location but User can't remove the filter on Page Level.
2. Cheque Voiding - Need to check and compare the Code for NAV16 & BC.
3. Exchange rate - Need to Increase the Length of the field Exchange Rate from 6 Decimal to 7 Decimal.
4. Cheque Print Alignment - Sandhya need to sit with the Client and fix it.

**Comment by TabrejAlam3K** [comment]
Hi Mr. @Manoj01021999 - As per our discussion We have discuss on the below points.

1. Permission Set - Collection Entry list page (Location Filter should be apply based on the User Setup Location).
2. Exchange Rate - Need to Increase the length of Decimal Place from 6 to 7 (In Collection Detail and Gen. Journal Line)

Please update the status of the Points.

CC : @audhi3k

**Comment by TabrejAlam3K** [comment]
Hi Mr. @Manoj01021999 - What is the status of the above points.

CC : @audhi3k @Shasthara @narayana3kt

**Comment by Manoj01021999** [comment]
@TabrejAlam3K 
i dont have git access for this, post getting the access only i can work on this 
@3ktech 
i need access for LKS

**Comment by TabrejAlam3K** [comment]
Hi Mr. @Manoj01021999 - Please update the current status of the Points.

CC : @audhi3k

**Comment by 3ktech** [comment]
> 2\. Exchange Rate - Need to Increase the length of Decimal Place from 6 to 7 (In Collection Detail and Gen. Journal Line)

hi @TabrejAlam3K , the customization is requested in standard page where the field property is not extendible. hence this is not possible. i would like to understand why such change is required? and what is the severity of business case for this request.
CC: @audhi3k , @Manoj01021999

**Comment by TabrejAlam3K** [comment]
Hi Mr. @3ktech - Changes is required because they want to handle the difference amount based on the decimal places.

Ms. Sandhya had done this changes earlier for the same requirement (from 2 decimal to 6 decimal).

CC : @audhi3k @Manoj01021999

**Comment by TabrejAlam3K** [comment]
I have created the separate ticket for open issues and closing this ticket.
